File: tks-examples/splitstring_benchmark.py
import time;

ms = round(time.time()*1000.0)

i = 0
for _ in range(1000):
  for _ in range(1000):
    s = "one two three four_and_a_half";
    for t in s.split():
      i += 1

ms = round(time.time()*1000.0) - ms
print(f"ms={ms}")

print(f"i={i}")

File: tks-examples/splitstring_benchmark.pl

my $i=0;
for($j=0; $j<1000; $j++)
{
    for($k=0; $k<1000; $k++)
    {
	my $s="one two three four_and_a_half";
	my $t;
	foreach $t (split(/ /,$s))
	{
	    $i++;
	}
    }
}
print "i=$i\n";
